sw/device/sca/aes_serial/aes_serial.c

```diff
@@ -198,7 +203,13 @@ static simple_serial_result_t simple_serial_trigger_encryption(
     return kSimpleSerialError;
   }
 
-  // start timer to wake up Ibex after AES is done.
+  // Provide input data to AES.
+  aes_data_put_wait(plain_text);
+
+  // Arm capture trigger.
+  SS_CHECK(dif_gpio_write_all(&gpio, kGpioCaptureTriggerHigh) == kDifGpioOk);
```

Comment:
I guess the 40 cycle delay you mentioned above lets us set the trigger high before putting Ibex to sleep, right? If so, would you mind adding a comment?

Reply:
I will add a comment to clarify this.